crates/core/src/mock/mock_starknet.rs

```diff
@@ -9,6 +11,60 @@ pub struct StarknetRpcFixture {
     response: Value,
 }
 
+pub struct StarknetRpcFixtureBuilder {
+    fixture: StarknetRpcFixture,
+    request: Value,
+    response: Value,
+}
+
+impl StarknetRpcFixtureBuilder {
+    pub fn new(method: JsonRpcMethod) -> Self {
+        Self {
+            fixture: StarknetRpcFixture { method, params: Value::Null, response: Value::Null },
+            request: Value::Null,
+            response: Value::Null,
+        }
+    }
+
+    pub fn load_jsons(mut self) -> Self {
+        let clean_quotations = |s: &str| s.replace('\"', "");
```

Comment:
This can be improved. The problem with the current solution is that for example, if you want to use a fixture for `starknet_call`, the file used will always be the same, so you can't mock different calls.